src/components/Expense.js

```diff
@@ -0,0 +1,27 @@
+const Expense = ({ expense, removeExpense }) => {
+  return (
+    <li className="container list-group-item">
+      <div className="row">
+        <div className="col">
```

Comment:
is it possible to combine these class with `fs-5`?

Reply:
Do you mean is it possible to apply `fs-5` to the div instead of the span?

Reply:
i meant to the div with `col`